FinSearchAPI/Resources/services.xml

```diff
@@ -23,6 +23,10 @@
 
         <service id="fin_search_api.shopware_process" class="FinSearchAPI\ShopwareProcess">
 
+        </service>
+
+        <service id="fin_search_api.article_model_factory" class="FinSearchAPI\BusinessLogic\FindologicArticleFactory">
```

Comment:
Isn't it possible to use `FindologicArticleModel` directly?

Reply:
I guess not because in this case we need to return new Instances of the `FindologicArticleModel`. See here https://developers.shopware.com/developers-guide/services/#calling-the-service